modules/index/companies-using-kedro/companies-using-kedro.module.scss

```
@@ -16,57 +16,57 @@
 }
 
 .inner {
-  background: $color-bg-grey;
-  flex-direction: column;
-  font-weight: 700;
-  line-height: 6rem;
   display: flex;
   margin: 0 auto;
-  max-width: $max-content;
-
-  @include breakpoint(small) {
-    font-size: 1.5rem;
-    line-height: 3rem;
-  }
 }
 
-.title {
-  @include h1;
-
-  margin-bottom: 6rem;
-  text-align: center;
-
-  @include breakpoint(small) {
-    margin-bottom: 2.5rem;
+.titleContainer {
+  white-space: nowrap;
 
-    padding: 0;
+  .title {
+    @include display-md;
+    line-height: 7rem;
+    font-weight: 500;
+    padding-right: 2em;
+    @include breakpoint(small) {
+      font-size: 2rem;
+      line-height: 7.25rem;
+      margin-bottom: 2.5rem;
+      padding-right: 1.5em;
+    }
   }
 }
 
-.list {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  grid-template-rows: 1fr 1fr 1fr 1fr;
-  justify-items: start;
-  list-style: none;
-  margin: 0;
-  padding: 0;
-
-  @include breakpoint(small) {
-    grid-template-columns: 1fr 1fr;
-  }
+.carouselWrapper {
+  overflow: hidden;
+  width: 100%;
+  display: flex;
+  position: relative;
```

I like to try and sort things (CSS styles, React props) alphabetically so it's easier for a reader to find what they're looking for.

```suggestion
  display: flex;
  overflow: hidden;
  position: relative;
  width: 100%;
```